src/site/content/en/design-system/css-utilities.njk

```diff
@@ -1,12 +1,97 @@
 ---
 title: 'CSS Utilities'
+permalink: '/design-system/css-utilities/index.html'
 ---
 
 {% extends "design-system.njk" %}
 
+{% set content %}
```

Comment:
There's almost certainly a better way to do this, but for now, it'll get this bit of documentation where it needs to be.

Reply:
Not sure I understand the problem but could you have a separate css-utilities markdown file that contains this content and uses css-utilities.njk as its layout?

Reply:
Ah yes! I've created a `design-system-documentation.njk` layout because there'll be a few pages like this. This fixes the syntax highlighting too 🙌